cmd/whereabouts_test.go

```diff
@@ -1341,15 +1359,15 @@ users:
 `)
 }
 
-func ipPool(ipRange string, namespace string, podReferences ...string) *v1alpha1.IPPool {
```

Comment:
I still think it is the responsibility of the caller to pass the `ipRange`. It is a code smell that we pass it the whole configuration when this just uses the range of the configuration. Especially one that assumes a particular index.

I would leave the function signature as it was, and pass it from the the test the range from which you want to build an `IPAMConfig` from.

Reply:
Thanks :)
I rolled back my changes in `ipPool` function and instead updated the test cases to pass the the range from the 1st element of `IPRanges` instead to "oldRange" (since it was removed).